Dockerfile

```diff
@@ -0,0 +1,18 @@
+ARG BUILD_IMAGE=eclipse-temurin:17-jdk-jammy
+ARG RUNTIME_IMAGE=eclipse-temurin:17-jre-jammy
+
+# Build stage
+FROM ${BUILD_IMAGE} as builder
+WORKDIR /app
+COPY .mvn/ .mvn
+COPY mvnw pom.xml ./
+RUN ./mvnw dependency:resolve-plugins dependency:go-offline -B
```

Comment:
@rafa-afonso 
Rafa, eu tive esse problema aqui:
```
> [builder 5/7] RUN ./mvnw dependency:resolve-plugins dependency:go-offline -B:
#12 0.273 /bin/sh: 1: ./mvnw: not found
```

Dando uma pesquisada, achei [essa página](https://stackoverflow.com/questions/61226664/build-docker-error-bin-sh-1-mvnw-not-found) do stackoverflow
- Mudando (no meu pc) pra LF funcionou

---

Não sei o quão frequente é mexer nesse `mvnw` mas é algo que pode enxer o saco (tipo, se eu mudo local, toda vez q commitar ele mudaria - sem contar q vai dar ruim pra quem clonar e for windows)

Uma solução que fizemos em CCR foi adicionar um `.gitattributes` ([ex](https://bitbucket.org/inspira_tecnologia/ccr-core-authentication-service/src/develop/.gitattributes))
- Baseado [nisso](https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings#refreshing-a-repository-after-changing-line-endings)


Acha que vale a pena?

Reply:
Vamos lá que passei por algumas etapas:
- Tentei usar o `.gitattributes` e não consegui fazer funcionar. Tentei clonar o repo em outra pasta, mas continuava vindo como CRLF.  
- Aí fui pela abordagem que a resposta do stack que você mandou mencionou, de usar o dos2unix e parece ter funcionado aqui. Mas não sei, me parece uma adição que não deveria ser necessária e faz o build demorar ainda mais :/
- Finalmente, aproveitei a dica do outro projeto que já usa uma imagem do docker do maven. Encontrei uma que parece ser exatamente o que a gente usava, mas com a adição oficial do maven. Assim a gente consegue usar o mvn direto, sem ter que lidar com essa questão do ./mvnw

Reply:
Importante é que funcionou hahaha